src/SumoCoders/FrameworkUserBundle/Entity/User.php

```diff
@@ -26,4 +26,9 @@ public function getRoles(): array
     {
         return ['ROLE_USER'];
     }
+
+    public function canSwitchTo(BaseUser $user): bool
```

Comment:
wouldn't this be a good default to be placed in the MultiUserBundle itself?

Reply:
Maybe yes, I have a look into this, just in BaseUser?

Reply:
@StijnVrolijk https://github.com/sumocoders/FrameworkMultiUserBundle/pull/65

Reply:
@StijnVrolijk Fixed 3fdb4d3